splunk/src/test/resources/log4j.properties

```diff
@@ -14,7 +14,6 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
-
```

Comment:
Why did you remove this line?

Reply:
> Why did you remove this line?

I did not intentionally modify it, it may have been deleted by mistake.